cms/static/js/certificates/views/signatory_details.js

```diff
@@ -6,7 +6,7 @@ define([ // jshint ignore:line
     'underscore.string',
     'backbone',
     'gettext',
-    'js/utils/templates',
+    'common/js/components/utils/templates',
```

Comment:
I would recommend that you don't bring this library over, because it is for managing static templates on the page. Instead I suggest that you convert the code in question to use RequireJS text because it makes it much simpler to consume, and because it will optimize better in production.